crm_lead_code/models/crm_lead.py

```diff
@@ -16,11 +16,14 @@ class CrmLead(models.Model):
         ("crm_lead_unique_code", "UNIQUE (code)", _("The code must be unique!")),
     ]
 
-    @api.model_create_multi
-    def create(self, vals_list):
-        for vals in vals_list:
-            if vals.get("code", "/") == "/":
-                vals["code"] = self.env.ref(
-                    "crm_lead_code.sequence_lead", raise_if_not_found=False
-                ).next_by_id()
-        return super().create(vals_list)
+    @api.model
```

Comment:
Don't remove the `model_create_multi` improvement

Reply:
thank you @pedrobaeza